lib/rules/font-family-generic-fallback/README.md

```diff
@@ -0,0 +1,39 @@
+# font-family-generic-fallback
+
```

Comment:
As described in the [associated issue](https://github.com/stylelint/stylelint/issues/2929), please change all references to the rule name to font-family-no-missing-generic.

Reply:
SGTM. As there seems to be [no objections](https://github.com/stylelint/stylelint/issues/2929#issuecomment-333479439) to the name proposal, @stoyan can you rename this `font-family-no-missing-generic`.